config/proto/service.proto

```diff
@@ -0,0 +1,48 @@
+/*
+  This proto defines the service-level config for Hermes.
+  This is also the external service API for Hermes.
+*/
+syntax = "proto3";
+
+package hermes;
+
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto";
+// This defines the (external) service-level config for Hermes.
+// It identifies what gRPCs Hermes will support.
+service Hermes {
+
+  // Start monitoring a new storage system.
+  rpc StartMonitoringStorageSystem(HermesProbeRequest) returns (HermesProbeResponse) {}
+
+  // Stop monitoring a storage system that is currently being monitored.
+  rpc StopMonitoringStorageSystem(StopMonitoringSystemRequest) returns (StopMonitoringSystemResponse) {}
+
+  // Lists the storage systems being monitored at the moment.
+  rpc ListMonitoredStorageSystems(ListMonitoredSystemsRequest) returns (ListMonitoredSystemsResponse) {}
+}
+// This is used for starting monitoring a new storage system using a Hermes probe.
+message HermesProbeRequest {
+  HermesProbeDef probe_config = 1;
+}
+// This is an empty response message for the probe response.
+// An exit status code may be added to probe responses, and included here.
+message HermesProbeResponse {}
+
+// A target must be specified so that Hermes knows which target to stop monitoring.
+message StopMonitoringSystemRequest {
+  repeated targets.TargetsDef target = 1;
```

Comment:
Ditto here and in the rest all other TargetsDef mentions.
```suggestion
  repeated TargetsDef target = 1;
```

Reply:
Amended.